azure-pipelines.yml

```diff
@@ -260,6 +260,26 @@ jobs:
           fi
         name: out
 
+  - job: check_perf_test
+    pool:
+      name: linux-pool
+    condition: eq(variables['Build.Reason'], 'IndividualCI')
+    steps:
+    - bash: |
+        TEST_SHA=$(cat ci/cron/test_sha)
+        LAST_CHANGES=$(git log -n1 --format=%H daml-lf/scenario-interpreter/src/perf)
+        CURRENT_SHA=$(git rev-parse HEAD)
```

Comment:
This only runs on "individual CI", i.e. not PRs, which means
- It does not need to worry about Azure running merge commits; the current commit really is the one we're running for.
- This is not tested as part of the PR checks. 🙏 driven development.